.github/workflows/linters.yml

```diff
@@ -54,3 +54,6 @@ jobs:
       - name: Check node_modules existence
```

Comment:
- [ ] ESLint is not properly configured, and hence not showing among the linter checks on your Pull Request. Can you endeavor to fix it pls?

**ESLint missing:**
![image](https://user-images.githubusercontent.com/12745474/211143654-01c36942-2f44-41e1-8314-09be519b84e0.png)